templates/gitlab-manager-runner-roles.yaml

```diff
@@ -33,12 +33,11 @@ Parameters:
       - 't3a.large'
       - 't3a.xlarge'
       - 't3a.2xlarge'
-      - 'c5d.large'
-      - 'c5d.xlarge'
-      - 'c5d.2xlarge'
       - 'r5a.large'
       - 'r5a.xlarge'
       - 'r5a.2xlarge'
+      - 'r5a.4xlarge'
+      - 'r5a.8xlarge'
     Default: 't3a.micro'
```

Comment:
maybe update to the same instance type as you are going to use for staging and prod?

Reply:
Sure. We don't really need to list `r5a.8xlarge` in there.

Reply:
`r5a.8xlarge` has been removed.